### PR TITLE
new/Create a dashboard view

### PR DESCRIPTION
**Pull request by aspedrosa**
Fixes [bioinformatics-ua/montra-pvt#1967](https://github.com/bioinformatics-ua/montra-pvt/issues/1967)

When opening the database dashboard on the EHDEN Portal of a database that has not yet uploaded Achilles files into the platform the following will appear:
image

As the issue at the top mentions, a message should be presented instead of rendering the dashboard with "no results" on the charts.

## Implementation
Created a new view/endpoint `uploader/[db hash]/dashboard` which checks
1) if the data source exists
2) if it exists, checks if any upload was done.

If one of these conditions fails, the following simple page will render:
image
else the view/endpoint will redirect to Superset's database dashboard

from `dashboards.ehden.eu/uploader/[db hash]/dashboard` to `https://superset.ehden.eu/superset/dashboard/database-level-dashboard/?standalone=1&preselect_filters=%7B%2269%22:%7B%22acronym%22:[%22DB_HASH%22]%7D%7D`.

I tested on an iframe and the redirect works nicely.

Note: github actions errors addressed on #243

### CODE REVIEW

File: docker/docker-compose-dev.yml [hunk 1]
```diff
@@ -110,11 +110,12 @@ services:
   dashboard_worker:
     build: *dashboard-build
     environment: *dashboard-environment
-    command: celery -A dashboard_viewer worker -Ofair -l INFO
+    command: celery -A dashboard_viewer worker -Ofair -l DEBUG
```
Comment (joaorafaelalmeida) [inline docker/docker-compose-dev.yml:113]: @aspedrosa DEBUG?

File: docker/docker-compose-dev.yml [hunk 2]
```diff
@@ -43,7 +43,7 @@ x-dashboard-environment: &dashboard-environment
   REDIS_CELERY_DB: 1
   REDIS_CONSTANCE_DB: 2
   SECRET_KEY: ${DASHBOARD_VIEWER_SECRET_KEY}
-  DASHBOARD_VIEWER_ENV: ${INSTALLATION_ENV}
+  DASHBOARD_VIEWER_ENV: development
```
Comment (joaorafaelalmeida) [inline docker/docker-compose-dev.yml:46]: @aspedrosa hardcoded?

File: dashboard_viewer/dashboard_viewer/settings.py [hunk 3]
```diff
@@ -275,6 +276,21 @@
...
+        "Host of the target superset installation. Used to redirect to the dashboard of a database",
+        str,
+    ),
+    "DATABASE_DASHBOARD_IDENTIFIER": (
+        "database-level-dashboard",
+        "Identifier of the database dashboard on the Superset installation.",
+        str,
+    ),
+    "DATABASE_FILTER_ID": (
+        69,
```
Comment (joaorafaelalmeida) [inline dashboard_viewer/dashboard_viewer/settings.py:290]: @aspedrosa why 69?

**Review by joaorafaelalmeida — CHANGES_REQUESTED** [review]

File: docker/docker-compose-dev.yml (hunk 1 above)
Comment (aspedrosa) [reply docker/docker-compose-dev.yml:113]: This is the development docker-compose file. Note the file name.

File: docker/docker-compose-dev.yml (hunk 2 above)
Comment (aspedrosa) [reply docker/docker-compose-dev.yml:46]: This is the development docker-compose file. Note the file name.

File: dashboard_viewer/dashboard_viewer/settings.py (hunk 3 above)
Comment (aspedrosa) [reply dashboard_viewer/dashboard_viewer/settings.py:290]: This is the id of the Datasource filter chart present on our Database-Level Dashboard.

**Review by joaorafaelalmeida — APPROVED** [review]

**Pull request merged**